### PR TITLE
UI

### PR DESCRIPTION
**Pull request by kierancassel**

### CODE REVIEW

File: PixabaySearch/GalleryViewController.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    var coordinator: AppCoordinator?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        galleryCollection.dataSource = self
+        galleryCollection.reloadData()
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 12
```
Comment (AdamKhanITC) [inline PixabaySearch/GalleryViewController.swift:21]: Create dummy function in view model to return 12

File: PixabaySearch/GalleryViewController.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = galleryCollection.dequeueReusableCell(withReuseIdentifier: "galleryCell", for: indexPath) as? GalleryCollectionViewCell
+        else { return UICollectionViewCell() }
+        cell.imageView.backgroundColor = UIColor(named: "Blue")
```
Comment (AdamKhanITC) [inline PixabaySearch/GalleryViewController.swift:27]: Function to set cell data should be in the view model

File: PixabaySearch/SearchViewController.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+class SearchViewController: UIViewController {
+    @IBOutlet weak var searchTextField: UITextField!
+    var coordinator: AppCoordinator?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    @IBAction func searchButtonPressed(_ sender: Any) {
```
Comment (kierancassel) [inline PixabaySearch/SearchViewController.swift:18]: Validate input (disallow empty text field)

File: PixabaySearch/SearchViewController.swift [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        coordinator?.search()
```
Comment (kierancassel) [inline PixabaySearch/SearchViewController.swift:19]: Pass search term through coordinator

File: PixabaySearch/AppCoordinator.swift [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    
+    func start()-> UINavigationController {
+        let sb = UIStoryboard(name: "Main", bundle: nil)
+        navController = sb.instantiateViewController(withIdentifier: "navController") as? UINavigationController
+        let vc = navController.viewControllers.first as? SearchViewController
+        vc?.coordinator = self
+        return navController
+    }
+    
+    func search() {
```
Comment (kierancassel) [inline PixabaySearch/AppCoordinator.swift:22]: Take search term and use for initialising gallery view model

**Review by AdamKhanITC — APPROVED** [review]

**Pull request merged**